std_msgs/ColorRGBA color    # RGB values [0-1] for all LEDs
int32  blinks               # Number of blinks (1 blink = 1 time on + 1 time off)
float32 duration_on         # in [s]
float32 duration_off        # in [s]
int32 start_led             # First LED in the stripe where LEDs will blink
int32 end_led               # Last LED in the stripe where LEDs will blink
---
int32 blinks_left           # Number of blinks left until the goal is completed (change this to some decent feedback)
---
int32 blinks_left           # Number of blinks left until the goal is completed
